add2cal/add2cal.py

```diff
@@ -29,14 +29,17 @@ def __init__(
         end,
         description,
         location,
-        timezone
+        timezone,
+        attendee
     ):
         self.start_datetime = start
         self.end_datetime = end
         self.event_title = title
         self.event_location = location
         self.timezone = timezone
         self.event_description = description
+        self.attendee_name = attendee[0] if attendee else None
```

Comment:
Do we know that `None` (rather than an empty string) will work in the places this data gets passed below?

Reply:
Fixed